tools/export.py

```diff
@@ -204,6 +207,14 @@ def main(args):
     fake_data = get_fake_input(cfg, device=device)
 
     # BEGIN nncf part
+    alt_ssd_export = getattr(args, 'alt_ssd_export', False)
+    if (is_checkpoint_nncf(args.checkpoint) and not cfg.get('nncf_config')
+        and not alt_ssd_export):
```

Comment:
Why does `alt_ssd_export` affect config reading part?

Reply:
Because sometimes in our tests export.py is called for nncf-compressed model.
Just because in our OTE tests alt_ssd_export is called for all models, even for NNCF-based (and it is hard to distinct is the model NNCF or not in that point)

If alt_ssd_export is called with nncf-model it is ok, if we do not wrap it by NNCF (although most probably quality is bad, but nobody cares)

Reply:
So, what will happen if we omit this condition here?

Reply:
The model will be wrapped by NNCFNetwork class, then openvino export will started with alt_ssd_export=True, and then the following assert will happen:
```python
        assert isinstance(model, detectors.SingleStageDetector)
```
-- it will through exception, because we will have NNCFNetwork model which wrap the original model, but it is not an instance of `SingleStageDetector`.

(And obviously if we remove this assert something wrong will be later, since we did not spend time to make work NNCFNetwork-wrapping with this branch of the code.)

Reply:
Cool. Does not that mean, that `alt_ssd_export` is incompatible with export of NNCF-compressed model? If so you need to check it explicitly and handle the case when nncf config is already a part of config.

Reply:
Oh, I see.
I will have to try to fix that on 2 levels above, in tests of OTE, not to start alt_ssd_export for NNCF-compressed models

Reply:
It's good to have this check here as well.

Reply:
Fixed!